image-swipe.d.ts

```diff
@@ -27,6 +32,9 @@ export class ImageSwipe extends ScrollView {
 
     public ios: any; /* UIScrollView */
     public android: any; /* android.support.v4.view.ViewPager */
+
+    public nextPage(): void;
```

Comment:
How about instead of having just next and previous page navigation we change this to `public scrollToPage(pageNumber: number, animated?: boolean)`? This will allow much broader cases than just +1 and -1 for pages. Also will allow the user to control whether this is animated or not.